### ISSUE TITLE
Problem when using the `urchin` library

### ISSUE THREAD

**Issue by LorenzoFiori** [ami-iit/human-model-generator#21]
While using the `urchin` [library](https://github.com/fishbotics/urchin) to add meshes for the links and muscles, I encountered a problem. Specifically, when adding the "material" to the code, it does not allow me to set the color in RGBa format:

Python code
```
material=Material(name="Link", color=OPT_COLOR_LINK_MESH),
```
with 
```
OPT_COLOR_LINK_MESH = [0.5, 0.8, 0.2, 1.0]
```
URDF code:
undesirable result (only name)
```
      <material name="Link"/>
```

The problem was promptly fixed by @traversaro, who generated a branch ([`traversaro-patch-2`](https://github.com/fishbotics/urchin/tree/traversaro-patch-2)) that has not yet been merged. This has resulted in an additional procedure for installation and tool dependencies (appropriately described in the README), which will be removed as soon as the branch is merged with the main.

**Comment by traversaro** [comment]
Just fyi, the change was merged and is now available in urchin's master branch, what we are waiting for is https://github.com/fishbotics/urchin/pull/23 for actually tagging a release.

**Comment by traversaro** [comment]
(there was a wrong link in the previous issue, now I fixed it).

**Comment by traversaro** [comment]
The fix was released as 0.0.29 and it is now available in conda-forge: https://github.com/conda-forge/urchin-feedstock/pull/1 . It will be available also on PyPI as soon as https://github.com/fishbotics/urchin/issues/24 is fixed.

**Comment by claudia-lat** [comment]
Thanks @traversaro
